sos/plugins/openstack_nova.py

```diff
@@ -27,9 +27,17 @@ class OpenStackNova(Plugin):
     profiles = ('openstack',)
 
     option_list = [("log", "gathers openstack nova logs", "slow", True),
+                   ("guru", "gathers guru meditation reports", "slow", True),
                    ("cmds", "gathers openstack nova commands", "slow", False)]
 
+    service_names = ['nova-compute', 'nova-api', 'nova-conductor',
+                     'nova-scheduler', 'nova-novncproxy']
+
     def setup(self):
+        if self.get_option("guru"):
+            for service in self.service_names:
+                self.check_ext_prog("kill -USR1 `pgrep %s`" % service)
```

Comment:
Open to suggestions here, this actually results in multiple reports being gathered as a USR1 signal is sent to multiple PIDs owned by the same service.